### DISCUSSION TITLE
Appreciation

### DISCUSSION THREAD

**Discussion by danieldavemena** [akanshSirohi/ShareX#46] [General]
This is the file sharing app I've been looking for! It is fast and reliable. Your app really deserve more recognition. 10/10

**Comment by akanshSirohi** [comment]
Thank you so much for your kind feedback! @danieldavemena

**Comment by totoio** [comment]
You have no idea how happy I am with this app! I use it on my android and pc too (WSA)... Very convenient and responsive AF... Thank you very much for making this awesome app.

**Comment by ghost** [comment]
Hello. Dear Akansh!
Your app deserves an award because you managed to create something beautiful and functional and FOSS which by itself should get praise.
I was stuck with "dooblou WiFi File Explorer" since 2019 (last updated) and now I can leave it behind knowing that there is a way better solution for file transferring/sharing.

**Comment by akanshSirohi** [comment]
@danieldavemena @totoio @iamnytskyi You guys can check this: https://github.com/akanshSirohi/ShareX/discussions/78
